src/main/java/com/elastic/support/util/RestExec.java

```diff
@@ -93,20 +93,13 @@ else if(status == 403){
       return responseString;
    }
 
-   public boolean execConfiguredQuery(String url, String query, String destination) {
+   public void execConfiguredQuery(String url, String query, String destination) {
 
       HttpResponse response = null;
       boolean ret = true;
```

Comment:
Weird formatting in this function now that you removed the `try` wrappers. And this variable is no longer needed / used.